From e7662fd8ce2e0fe50d7b1c65cb893fd33c3c6f99 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C5=81ukasz?= <lukasz@czak.pl>
Date: Thu, 7 May 2020 13:41:00 +0200
Subject: [PATCH 3/3] FIXME: Don't use vop_crtc_mode_valid

DSI screen mode is filtered out by clock tolerance check.
---
 drivers/gpu/drm/rockchip/rockchip_drm_vop.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/rockchip/rockchip_drm_vop.c b/drivers/gpu/drm/rockchip/rockchip_drm_vop.c
index 1c0424bba..0a50b230d 100644
--- a/drivers/gpu/drm/rockchip/rockchip_drm_vop.c
+++ b/drivers/gpu/drm/rockchip/rockchip_drm_vop.c
@@ -1411,7 +1411,7 @@ static void vop_crtc_atomic_flush(struct drm_crtc *crtc,
 }
 
 static const struct drm_crtc_helper_funcs vop_crtc_helper_funcs = {
-	.mode_valid = vop_crtc_mode_valid,
+	/* .mode_valid = vop_crtc_mode_valid, */
 	.mode_fixup = vop_crtc_mode_fixup,
 	.atomic_check = vop_crtc_atomic_check,
 	.atomic_begin = vop_crtc_atomic_begin,
-- 
2.26.2

